architecture/additional_concepts/admission_controllers.adoc

```diff
@@ -116,7 +116,10 @@ certain behavior, such as:
 - xref:../../admin_guide/managing_projects.adoc#limit-projects-per-user[Limiting Number of Self-Provisioned Projects Per User]
 ifdef::openshift-enterprise,openshift-origin[]
 - xref:../../install_config/build_defaults_overrides.adoc#install-config-build-defaults-overrides[Configuring Global Build Defaults and Overrides]
-- xref:../../admin_guide/scheduling/scheduler.adoc#controlling-pod-placement[Controlling Pod Placement]
+endif::[]
+ifdef::openshift-enterprise,openshift-origin[]
```

Comment:
@mburke5678 - either a bad rebase here or conflict resolution. If you see the commit just before this commit for the 3.6 branch for this file, you will notice that the endifs are moved to the end of the file: https://github.com/openshift/openshift-docs/commit/a887a3dad89535daca194bc79e79d8c597f220b1#diff-79d834bf4b2464d1aec53f03bc68bca8R128.